designing-reports/report-designer-tools/desktop-designers/visual-studio-report-designer/overview.md

```diff
@@ -161,7 +161,7 @@ You may add custom code to the _ReportName_ type in the `ReportName.cs` file, wh
 
 ## Troubleshooting
```

Comment:
Do we need a detailed explanation about the supported VS versions in the Troubleshooting section?

Reply:
I removed this